simulate-travis.py

```diff
@@ -97,6 +97,12 @@
                 then we run independent mulled-build tests on
                 a successfully-built recipe. Use this argument to disable this
                 behavior''')
+ap.add_argument('--disable-local', action='store_true', help='''If you are on
```

Comment:
I though about using a `make` style syntax: `./simulate-travis.py [target [target...]]` where `target` is one of `native`, `docker`, `linux`, `all`, `osx` and `mulled`:

host/target|native|docker|mulled|linux|osx|all (default)
-|-|-|-|-|-|-
linux|native|FAIL|FAIL|native|FAIL|native
osx|native|FAIL|FAIL|FAIL|native|native
linux+docker|native|docker|mulled|mulled|FAIL|mulled
osx+docker|native|docker|mulled|mulled|native|mulled+native

Or, as if-else:

`native` :: build directly
`docker` :: build in `docker`, FAIL if `no docker`
`mulled` :: build in `docker` and run `mulled` tests, FAIL if `no docker`

`linux` :: if `docker` is available, `mulled`; otherwise if on `linux` `native` and if on `osx` FAIL
`osx` :: if on `osx` `native`; otherwise FAIL

`all` (default) :: if on `linux`, `linux`; if on osx `native` _and_ if `docker` is available `mulled`